src/src/Commands/Settings.cpp

```diff
@@ -92,7 +93,7 @@ const __FlashStringHelper * Command_Settings_Print(struct EventStruct *event, co
 	serialPrintln(F("System Info"));
 	serialPrint(F("  IP Address    : ")); serialPrintln(NetworkLocalIP().toString());
 	serialPrint(F("  Build         : ")); serialPrintln(String(get_build_nr()) + '/' + getSystemBuildString());
-	serialPrint(F("  Name          : ")); serialPrintln(Settings.Name);
+	serialPrint(F("  Name          : ")); serialPrintln(Settings.getUnitname());
```

Comment:
Not sure if this is still ever used by anyone, let alone whether they would do it in some script. So this probably won't change anything....